custom_components/foxess_modbus/entities/xx1_aux_entity_descriptions.py

```diff
@@ -46,6 +50,18 @@
         scale=0.001,
         validate=[Range(0, 10000)],
     ),
+    ModbusIntegrationSensorDescription(
+        key="pv1_power_total",
+        address=11002,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL,
+        native_unit_of_measurement="kWh",
+        integration_method="left",
+        name="PV1 Power Total",
+        round_digits=2,
+        source_entity="sensor.pv1_power",
```

Comment:
Will this work where the inverter has a friendly name prefix (which dual inverter setups need)?

Reply:
Yes, there's code above which copes with that

Reply:
Unfortunately not - the entity id (`sensor.pv1_power`) is the only unique identifier of this sensor, if it's renamed all bets are off!

Reply:
I guess if we could get a reference to the ModbusSensorEntity instance representing the PV1 sensor, we should be able to query the renamed entity_id from it? Since HA must be able to link the two. Maybe we can event query it for a link between original and renamed entity id somehow, since it must know. I've no idea how to go about it though. I suggest we get this in as is and worry about that later!